package.json

```diff
@@ -51,6 +51,7 @@
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-flow": "7.0.0",
     "@changesets/cli": "^1.3.0",
+    "ajv": "^8.5.0",
```

Comment:
The current tests runs against ajv 6, it now runs against latest.

In my fork I have opted to test against all versions declared in `peerDependencies` instead, see [test](https://github.com/ext/better-ajv-errors/blob/master/src/__tests__/index.js)